src/components/accountDisplay.tsx

```diff
@@ -14,25 +15,33 @@ export class AccountDisplay extends React.Component<AccountDisplayProps, {}> {
     super(props);
   }
 
+  public handleBackClick = () => {
+    this.props.onBackClick(null);
+  }
+
   public render() {
-    if (!this.props.acc){
+    if (!this.props.acc) {
       return <div>No Account Found</div>;
     }
 
     return (
-      <Card title={this.props.acc.name}>
```

Comment:
you don't need to change any of the display of this component for the assignment